### ISSUE TITLE
PseudoElement is not working as expected

### ISSUE THREAD

**Issue by amree** [brigade/scss-lint#827]
I was expecting `PseudoElement` to give me warning, but apparently it didn't. 

```
// style.css
p:before {
  content: '>';
}
```

I use the default config (which set `PseudoElement` to `true`) and ran this command:

```
scss-lint -c default.yml style.css
```

But it didn't give any message. Is this a bug or did I do somthing wrong?

**Comment by sds** [comment]
Hey @amree,

You've likely done something wrong, as I wasn't able to reproduce. Using the following configuration:

```yaml
linters:
  PseudoElement:
    enabled: true
```

...running against the following SCSS:

```scss
p:before {
  content: '>';
}
```

...I received the following output:

```
test.scss:1:1 [W] PseudoElement: Begin pseudo elements with double colons: `::`
```

Please ensure you are using the latest version of `scss-lint` (0.49.0 at time of writing) and that your configuration is not malformed. 

Hope this helps!
